examples/nim.cfg

```diff
@@ -1,44 +1,103 @@
---passC:"-I../../wxWidgets-3.0.2/lib/gcc_lib/mswu"
+@if vcc:
+--passC:"/I..\\..\\wxWidgets-3.0.2\\include"
```

Comment:
These building instructions need to move in to wx.nim so that other programs which use them get them too.